client/src/containers/signup/SignupForm.js

```diff
@@ -20,9 +20,11 @@ import {
   Form,
   Label,
   Input,
-  Transition } from 'semantic-ui-react';
+  Transition, Icon } from 'semantic-ui-react';
```

Comment:
`Icon` should be on a new line. Preferably alphabetized.

Reply:
addressed